src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -69,8 +69,8 @@ public static void UnicodePropertyNames()
             }
 
             {
-                // Max property length in C# is 511; we want to go over StackallocThreshold=256 to force a pooled allocation. This property is 400 chars and 401 bytes.
-                const int charsInProperty = 350;
+                // We want to go over StackallocThreshold=256 to force a pooled allocation, so this property is 200 chars and 400 bytes.
```

Comment:
Would this work to test all branches or do we want the chars to be > 256 as well?

Reply:
It hits the code that allocs... Really only need 256\6 = 43 chars since the expansion factor is 6 (MaxExpansionFactorWhileEscaping = 6)